crates/router/src/configs/settings.rs

```diff
@@ -623,7 +623,7 @@ pub struct Proxy {
     pub http_url: Option<String>,
     pub https_url: Option<String>,
     pub idle_pool_connection_timeout: Option<u64>,
-    pub bypass_proxy_urls: Vec<String>,
+    pub bypass_proxy_hosts: Option<String>,
```

Comment:
Not using a `Vec` or `HashSet` here (and splitting the string on commas), we pass the string directly to [`NoProxy::from_string()`](https://docs.rs/reqwest/0.12.11/reqwest/struct.NoProxy.html#method.from_string), which handles the splitting by comma and trimming whitespaces.